packages/website/components/Topbar.tsx

```diff
@@ -3,14 +3,14 @@ import tokens from '@contentful/f36-tokens';
 import { Subheading, Flex } from '@contentful/f36-components';
 import { css } from 'emotion';
 import Link from 'next/link';
-import { DocSearch } from './DocSearch';
+// import { DocSearch } from './DocSearch';
 
 export const TopbarHeight = '70px';
 
 const styles = {
   header: css`
     display: grid;
-    grid-template-columns: 320px auto 320px;
+    grid-template-columns: 3fr 7fr 2fr;
```

Comment:
This is working great, but I think you need to change the navListItem class to use `marginRight` instead of `marginLeft`

![Screenshot 2022-01-13 at 13 42 46](https://user-images.githubusercontent.com/6597467/149332418-1fe8f1b5-cffc-43ed-b77d-c0a517267dc6.png)